custom/build.gradle.kts

```diff
@@ -3,7 +3,8 @@ plugins {
 }
 
 val instrumentations = listOf<String>(
-  ":instrumentation:openai-client-instrumentation"
+  ":instrumentation:openai-client-instrumentation:instrumentation-0.13.0",
+  ":instrumentation:openai-client-instrumentation:instrumentation-latest"
```

Comment:
[question] where does the `latest` comes from, is it a feature of the gradle dependencies to allow easily getting the latest ?